src/less/expressive-loader/expressive-loader.less

```diff
@@ -0,0 +1,210 @@
+@import "../variables/variables.less";
+
+:root {
+    --expressive-loader-message-fadein-duration: 833ms;
+    --expressive-loader-message-fadeout-duration: 400ms;
+    --expressive-loader-scroll-lines-duration: 20s;
+    --expressive-loader-scale-line-duration: 0.7s;
+}
+
+.expressive-loader {
+    align-items: stretch;
+    display: flex;
+    flex-direction: column;
+    gap: var(--spacing-300);
+    justify-content: flex-end;
+    margin: 0 auto;
+    max-width: 100%;
+    width: 375px;
```

Comment:
I've swapped `max-width` and `width` here; 375px comes from the design spec. Tested it in ebayui-core Storybook, too, where it appears horizontally scrunched since the parent component doesn't have a specified width. I wonder if I should also add a min-width?

`gap` sets the vertical spacing between the messages and the progress bar.

> It looks like on smaller screens the loader doesn't expand to fill the container. It feels like it should. Is that by design?

The design spec calls for 16px padding on either side of the progress bar.

> Also, the reduced motion version seems to have a different width/max-width behavior. Those should probably be consistent.

This should be fixed now!